InvoiceMaker/Models/WorkLineItem.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace InvoiceMaker.Models
+{
+    public class WorkLineItem : ILineItem
+    {   
+        public decimal Amount { get; private set; }
+        public string Description { get; private set; }
+        public DateTimeOffset When { get; private set; }
```

Comment:
Instead of defining properties with setters, the intention was to store the `WorkDone` item passed through the constructor into a private field and use getter-only properties.